imaging/ml/ml_codelab/breast_density_cloud_ml.ipynb

```diff
@@ -1,911 +1,930 @@
+    "    --module-name \"scripts.trainer.model\" \\\n",
+    "    --package-path scripts \\\n",
+    "    --staging-bucket $3 \\\n",
+    "    --region $1 \\\n",
+    "    -- \\\n",
+    "    --bottleneck_dir \"$2/record\" \\\n",
+    "    --training_steps $4 \\\n",
+    "    --learning_rate $5 \\\n",
+    "    --export_model_path $6"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "_mVUV9GlNI3e"
+   },
+   "source": [
+    "You can monitor the status of the training job by running the following command. The job can take a few minutes to start-up."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "qgSKpZYCNPVD"
+   },
+   "outputs": [],
+   "source": [
+    "!gcloud ai-platform jobs describe breast_density"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "os3RAC17Nsjo"
+   },
+   "source": [
+    "When the job has started, you can observe the logs for the training job by executing the below command (it will poll for new logs every 30 seconds).\n",
+    "\n",
+    "As training progresses, the logs will output the accuracy on the training set, validation set, as well as the [cross entropy](http://ml-cheatsheet.readthedocs.io/en/latest/loss_functions.html). You'll generally see that the accuracy goes up, while the cross entropy goes down as the number of training iterations increases.\n",
+    "\n",
+    "\n",
+    "Finally, when the training is complete, the accuracy of the model on the held-out test set will be output to console. The job can take a few minutes to shut-down."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "RbqGVOvoOPkc"
+   },
+   "outputs": [],
+   "source": [
+    "!gcloud ai-platform jobs stream-logs breast_density --polling-interval=30"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "j2NI-ILU9HoN"
+   },
+   "source": [
+    "### Export Trained Model for Inference in Cloud ML Engine\n",
+    "\n",
+    "Cloud ML Engine can also be used to serve the model for inference.  The inference model is composed of the pre-trained Inception V3 checkpoint, along with the classification layers we trained above for breast density. First we set the inference model name and version."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "HvniHAbZGeVp"
+   },
+   "outputs": [],
+   "source": [
+    "model_name = \"breast_density\" # @param\n",
+    "version = \"v1\" # @param\n",
+    "\n",
+    "# The full name of the model.\n",
+    "full_model_name = \"projects/\" + project_id + \"/models/\" + model_name + \"/versions/\" + version"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "executi
```

Comment:
Same thing here what happened to installing kubectl?

Reply:
kubectl already present in environment